docker/Dockerfile

```diff
@@ -104,11 +107,12 @@ COPY --from=us-docker.pkg.dev/zealous-zebra/zebra/lightwalletd /opt/lightwalletd
 # This is the caching Docker layer for Rust!
 #
 # TODO: is it faster to use --tests here?
-RUN cargo chef cook --release --features "sentry ${TEST_FEATURES} ${FEATURES}" --workspace --recipe-path recipe.json
+RUN cargo chef cook --release --features "${TEST_FEATURES} ${FEATURES}" --workspace --recipe-path recipe.json
```

Comment:
> Could we add 'sentry' in to this features argument in release-binaries.yml:

Done!

> Nitpick:

It's meant to be separate so it can be overridden separately. I'll add a note.